task1.py

```diff
@@ -0,0 +1,6 @@
+print("Multi-paradigm programming language:Task 1")
```

Comment:
use single quotes for string literals

Reply:
I would recommend you to divide your code for logical blocks by blank lines.
for instance in such code you have logical block where you print information about yourself.
the other one is block with printing teacher name.
and the last one is math calculation